examples/meta/src/gaussian_process/classifier.sg

```diff
@@ -18,6 +17,7 @@ ConstMean mean_function()
 
 #![create_instance]
 SoftMaxLikelihood gauss_likelihood()
+gauss_likelihood.put("seed", 1)
```

Comment:
why would a likelihood have the need for a rng?

Reply:
@karlnapf it has a  `mc_sampler` method that creates a `GaussianDistribution` object that needs to be seeded